db/migrate/20180914132709_convert_accepting_to_enum.rb

```diff
@@ -0,0 +1,35 @@
+class ConvertAcceptingToEnum < ActiveRecord::Migration[5.1]
+  def up
+    # NOTE: Wanted to RAISE a DB error on unknown value,
+    #       but seems Rails/AR doesn't support that
+    # ... ELSE RAISE EXCEPTION 'UNKNOWN ACCEPTING VALUE IN ROW (ID %): %', id, accepting
+    execute <<-DDL
+      CREATE TYPE shelter_accepting_value AS ENUM (
+        'true', 'false', 'unknown'
+      );
+
+      ALTER TABLE shelters
+      ALTER COLUMN accepting TYPE shelter_accepting_value
+      USING CASE accepting
+            WHEN TRUE THEN 'true'::shelter_accepting_value
+            WHEN FALSE THEN 'false'::shelter_accepting_value
+            WHEN NULL THEN 'unknown'::shelter_accepting_value
```

Comment:
This line will never execute because `WHEN NULL` doesn't evaluate to true for whatever reason.  Recommend removing this line to ensure there aren't any unexpected behaviors.

You can verify by running this script:
```sql
CREATE TEMP TABLE converttest (id serial primary key, it boolean default null);
INSERT INTO converttest (it) VALUES (true), (true), (false), (default), (true), (default), (false);
ALTER TABLE converttest
ALTER COLUMN it TYPE text
USING CASE it
  WHEN true THEN 'yes'
  WHEN false THEN 'no'
  WHEN NULL THEN 'unknown'
  ELSE 'error'
END;
SELECT * FROM converttest;
```